plugins/reporters/spdx/src/main/kotlin/SpdxDocumentModelMapper.kt

```diff
@@ -70,7 +70,11 @@ internal object SpdxDocumentModelMapper : Logging {
                 ortResult
             )
 
-            ortResult.getDependencies(project.id, 1).mapTo(relationships) { dependency ->
+            ortResult.getDependencies(
+                id = project.id,
```

Comment:
Commit message nit: It's not "a type parameter" but "typed parameters" as also `id` is named now.